### DISCUSSION TITLE
Unable to resolve server information | No such host is known

### DISCUSSION THREAD

**Discussion by Fasapokhai** [dataplat/dbatools#8770] [General] [answered]
I am not able to remote to servers in my domain from my desktop .
See error below.
' Unable to resolve server information | No such host is known'.
I have the server set on remote assigned.
Please avdvise.

**Comment by niphlod** [comment, marked as answer]
"remote to servers in my domain" is kinda open to interpretations .... what command are you trying to run ?
it seems that your desktop is unable to resolve the host you're pointing to.

**Reply by Fasapokhai** [reply]
Test-DbaConnection sqlinstance servername

**Reply by Fasapokhai** [reply]
I am preparing to start using powershell in this domain for dba activities

**Reply by niphlod** [reply]
[Test-DbaConnection](https://docs.dbatools.io/Test-DbaConnection) targets only SQL instances ........ so you need to do just `Test-DbaConnection thenameofyourinstance`

**Reply by Fasapokhai** [reply]
Thank you ! 
Question answered !
